packages/firestore/src/core/sync_engine.ts

```diff
@@ -209,7 +200,7 @@ export interface SyncEngine {
  * This is useful to implement optional features (like bundles) in free
  * functions, such that they are tree-shakeable.
  */
-class SyncEngineImpl implements SyncEngine {
+export class SyncEngineImpl implements SyncEngine {
```

Comment:
Please don't export this class, as it leaks all the public members that are only public for consumers in this file.

Reply:
Done.